src/main.js

```diff
@@ -425,19 +426,7 @@ window.LinkedinToResumeJson = (() => {
                         ..._outputJsonLatest.basics,
                         ...formattedProfileObj
                     };
-                    /** @type {ResumeSchemaStable['languages'][0]} */
-                    const formatttedLang = {
-                        language: localeObject.language,
-                        fluency: 'Native Speaker'
-                    };
-                    _outputJsonStable.languages.push(formatttedLang);
-                    _outputJsonLatest.languages.push(formatttedLang);
                     resultSummary.sections.basics = 'success';
-
-                    // Also make sure instance defaultLocale is correct, while we are parsing profile
```

Comment:
Removing this section is problematic - tracking `defaultLocale` is important for multi-lingual profile support.